src/sonic-config-engine/tests/sample_output/t1-smartswitch.json

```diff
@@ -379,6 +375,38 @@
 		}
 	},
 	"INTERFACE": {
+		"Ethernet0":{},
```

Comment:
>Ethernet0 [](http://example.com/codeflow?start=3&length=9)

Does this PR enforce YANG validation? I am worried about future regression.

Reply:
Will use seperate PR